storage/remote/otlptranslator/prometheusremotewrite/metrics_to_prw.go

```diff
@@ -191,6 +194,26 @@ func (c *PrometheusConverter) FromMetrics(ctx context.Context, md pmetric.Metric
 	return annots, errs
 }
 
+func getTranslatorMetricType(metric pmetric.Metric) translator.MetricType {
+	var translatorMetricType translator.MetricType
+	if metric.Type() == pmetric.MetricTypeGauge {
```

Comment:
why not use a switch for this?

Reply:
There are cases where I check the OTLP type and also if it is monotonic or not. Can switch check two conditions like that?

Reply:
ahh I see I see. all good